modules/va_facilities/app/controllers/va_facilities/v0/facilities_controller.rb

```diff
@@ -68,18 +68,33 @@ def set_default_format
       private
 
       def validate_params
```

Comment:
Not sure if the existing specs handle the error cases well, but it might be nice to add a few specs that exercise the new validation for lat and long presence and float-ness ;)